<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dynamicprogramming</class>
 <widget class="QDialog" name="dynamicprogramming">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>131</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="SecondTaskLabel">
      <property name="text">
       <string>SecondTask</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>30</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="InputFromFileMatrixButton">
      <property name="text">
       <string>InputFromFileMatrix</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SecondTaskButton">
      <property name="text">
       <string>Solve</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="SecondTaskAnswearEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="FirstTaskLabel">
      <property name="text">
       <string>FirstTask</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="NumberOfElemLabel">
      <property name="text">
       <string>Number Of Elements</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="NumberOfElemEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="ElementsLabel">
      <property name="text">
       <string>Elements</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="ElementsEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="FirstTaskButton">
      <property name="text">
       <string>Solve</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="FirstTaskAnswearLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
